src/org/robockets/steamworks/commands/Shoot.java

```diff
@@ -0,0 +1,37 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.commons.YAxisRelativeDirection;
+import org.robockets.steamworks.Robot;
+
+/**
+ * @author Jake Backer
+ */
+public class Shoot extends Command {
+
+    public Shoot() {
+
+    }
+
+    protected void initialize() {
+
+    }
+
+    protected void execute() {
+        Robot.shooter.spinUp();
```

Comment:
Shouldn't this line be in `initialize` instead? To spin up a motor means to start it from rest and you only do that once. Or should `spinUp()` be renamed?